region/norfair/west/Ice Beam Gate Room.json

```
@@ -310,15 +310,50 @@
       "link": [2, 4],
       "name": "Mockball",
       "entranceCondition": {
-        "comeInRunning": {
-          "minTiles": 11,
-          "speedBooster": "any"
+        "comeInWithMockball": {
+          "adjacentMinTiles": 11,
+          "remoteAndLandingMinTiles": [[8, 2], [7, 3]]
+        }
+      },
+      "requires": []
+    },
+    {
+      "link": [2, 4],
+      "name": "Mockball (Speed Booster, Short Landing)",
+      "entranceCondition": {
+        "comeInWithMockball": {
+          "remoteAndLandingMinTiles": [[9, 1]]
         }
       },
       "requires": [
-        "canMockball"
-      ],
-      "devNote": "FIXME Mockballing from the next room can also be done with smaller runways (like in the vanilla strat). See issue #63."
+        "SpeedBooster"
+      ]
+    },
+    {
+      "link": [2, 4],
+      "name": "Tricky Mockball",
+      "entranceCondition": {
+        "comeInWithMockball": {
+          "adjacentMinTiles": 9,
+          "remoteAndLandingMinTiles": [[7, 1], [6, 2], [5, 3]]
+        }
+      },
+      "requires": [
+        "canTrickyJump"
+      ]
+    },
+    {
+      "link": [2, 4],
+      "name": "Insane Mockball",
+      "entranceCondition": {
+        "comeInWithMockball": {
+          "adjacentMinTiles": 7.4375,
```

```suggestion
          "adjacentMinTiles": 10
```
I switched to console and was able to get it at 8 tiles.  If we wanted an `insane` version we could go with that number.  At 10 it felt like you didn't need the ultraprecise jump tap and was more like something I could do in a few tries.  Practicing these instant down presses felt like it has helped my shorthop mockballs.

A 1 frame button press and release, like in the video, is `insane` movement.

---

How about 9.4375 here? This would match the Grapple Beam Room runway, which in the past was a common connection with Ice Beam Gate and where I've seen many players do it successfully. After some practice, I find I can get it (at full speed) with only a few tries on average, so for Extreme tech a 9.4375 it would seem on the easy end; but considering the room could be heated, it makes sense not to push it too far.